src/components/calcite-tab-title/calcite-tab-title.tsx

```diff
@@ -129,15 +130,17 @@ export class CalciteTabTitle {
   //
   //--------------------------------------------------------------------------
 
-  @Listen("calciteTabChange", { target: "parent" }) tabChangeHandler(
+  @Listen("calciteTabChange", { target: "body" }) tabChangeHandler(
     event: CustomEvent<TabChangeEventDetail>
   ) {
-    if (this.tab) {
-      this.active = this.tab === event.detail.tab;
-    } else {
-      this.getTabIndex().then((index) => {
-        this.active = index === event.detail.tab;
-      });
+    if (this.parentTabNavEl === event.target) {
```

Comment:
I think you can combine these `if` statements to avoid double nesting. What do you think @macandcheese

Reply:
With an extra `if else`:

```
  const isParent = this.parentTabNavEl === event.target;
    if (isParent && this.tab) {
      this.active = this.tab === event.detail.tab;
    } else if (isParent) {
      this.getTabIndex().then((index) => {
        this.active = index === event.detail.tab;
      });
    }
```